server/src/test/java/WriteStoreMultithreadTest.java

```diff
@@ -0,0 +1,201 @@
+/*
+ *  Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import io.seata.core.exception.TransactionException;
+import io.seata.core.model.BranchStatus;
+import io.seata.core.model.GlobalStatus;
+import io.seata.server.session.BranchSession;
+import io.seata.server.session.GlobalSession;
+import io.seata.server.session.SessionCondition;
+import io.seata.server.session.SessionManager;
+import io.seata.server.store.FileTransactionStoreManager;
+import io.seata.server.store.TransactionStoreManager;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * The interface Session storable.
+ *
+ * @author lizhao
+ */
+public class WriteStoreMultithreadTest {
+    private static String vgroup = "vgroupMock";
+    private static String appname = "appnameMock";
+    private static String instname = "fescarMocK";
+    private static int per_thread_trx_num = 65535;
+    private static int threadNum = 16;
+    private static CountDownLatch countDownLatch = new CountDownLatch(threadNum);
+    public static void main(String[] args) throws Exception {
```

Comment:
please use @Test instead of main method,and some Asserts to verify it.

Reply:
Reference WriteStoreTest, it is not a unit test